engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/TopicSelectionStrategy.scala

```diff
@@ -0,0 +1,28 @@
+package pl.touk.nussknacker.engine.avro
+
+import cats.data.Validated
+import pl.touk.nussknacker.engine.avro.schemaregistry.{SchemaRegistryClient, SchemaRegistryError}
+
+import java.util.regex.Pattern
+
+trait TopicSelectionStrategy {
+
+  val schemaRegistryClient: SchemaRegistryClient
```

Comment:
I think this is not needed here? Let's either remove it, or put schemaRegistryClient to getTopics parameter (and make Strategy stateless)